frontend/src/features/admin-form/responses/common/utils/mrfSubmissionView.ts

```diff
@@ -17,15 +17,15 @@ export const getCurrentStepString = (
 /** Gets the business friendly string for MRF submission status. */
 export const getStatusFromWorkflowStatus = (
   workflowStatus: WorkflowStatus | undefined,
-) => {
+): MRF_STATUS | '' => {
   switch (workflowStatus) {
     case WorkflowStatus.COMPLETED:
     case WorkflowStatus.APPROVED:
     case WorkflowStatus.REJECTED:
       return MRF_STATUS.COMPLETED
     case WorkflowStatus.PENDING:
       return MRF_STATUS.PENDING
-    default:
+    case undefined:
```

Comment:
1. Should still have a default case for fall-through, recommend to utilise [exhaustive switch](https://www.notion.so/opengov/Use-exhaustive-switch-for-enums-e273c275a684492d8aef4789b4882906)
2. Check for undefined should be outside of switch case. Could be something before line 21.

Reply:
Thanks for sharing this -- it is a very cool usage of typescript